blackbox/docs/sql/information_schema.txt

```diff
@@ -27,63 +27,41 @@ settings like the ``number of shards`` or ``number of replicas``.
     ... ) partitioned by (date);
     CREATE OK, 1 row affected (... sec)
 
-::
-
-    cr> select table_schema, table_name, number_of_shards, number_of_replicas
```

Comment:
I would not remove this - or add at least one example how to query the information_schema.tables. @nslater what do you think?